doc/api/next_api_changes/deprecations.rst

```diff
@@ -376,3 +376,8 @@ NavigationToolbar2QT.parent
 This attribute is deprecated.  In order to access the parent window, use
 ``toolbar.canvas.parent()``.  Once the deprecation period is elapsed, it will
 also be accessible as ``toolbar.parent()``.
+
```

Comment:
this now lives in the 3.2 docs, but we need a not about it going away for real.

Reply:
ah...